omero/developers/JavaGateway.txt

```diff
@@ -0,0 +1,20 @@
+Java Gateway
+------------
+
+The Java :javadoc:`Gateway <omero/gateway/Gateway.html>` is a
+wrapper around the :zerocdoc:`Ice Java language mapping
+<display/Ice/Hello+World+Application>` and the :doc:`Modules/Api` which makes
+it easier to interact with an OMERO server in Java.
+
+The :javadoc:`Gateway <omero/gateway/Gateway.html>` is the central object
+for maintaining the connection to the server. 
+See :ref:`gatewayconnect`
+
+Functionality for interacting with the server is encapsulated into different
+:javadoc:`Facilities </omero/gateway/facility/package-summary.html>`
+For example using the :javadoc:`BrowseFacility <omero/gateway/facility/BrowseFacility.html>`
+to access Projects, Datasets, etc.: :ref:`gatewaybrowse`
```

Comment:
I don't know what the `:` is doing here and the ref looks rather odd with no clear explanatory text, maybe this needs  "see... for further information." or some such.